cairo/ethereum/cancun/utils/address.cairo

```diff
@@ -35,8 +36,7 @@ func to_address{range_check_ptr}(data: UnionUintU256) -> Address {
     }
 
     with_attr error_message("Type not valid") {
-        assert 0 = 1;
-        ret;
+        jmp raise.raise_label;
```

Comment:
why not invoking raise('type not valid')?

Reply:
you need to end a function by either a ret or a jmp or a dw